apps/system/lockscreen/js/lockscreen.js

```diff
@@ -541,6 +539,16 @@
     this._unlockingMessage = {};
   };
 
+  /**
+   *  Plays unlock sound if required
+   */
+  LockScreen.prototype.playUnlockSound = function(soundFileUrl) {
```

Comment:
I think it's weird when you are **locking** and you call a method named as 'play**Unlock**Sound', so I suggest to make it as two methods like `playUnlockSound` and `playLockSound` to specify which sound you're playing. Another advantage is you can encapsulate the sound file URL inside the method, rather to pass it as an argument while it's actually necessary to be that.